roles/network/tasks/install.yml

```diff
@@ -76,6 +76,45 @@
   include_tasks: roles/network/tasks/squid.yml
   when: squid_install and squid_installed is undefined
 
+- name: Link /usr/local/sbin/netwarn to {{ iiab_dir }}/scripts/netwarn
+  file:
+    src: "{{ iiab_dir }}/scripts/netwarn.sh"
+    dest: /usr/local/sbin/netwarn
+    mode: 0755
```

Comment:
`man chmod` says:

> chmod never changes the permissions of symbolic links; the chmod system call cannot change their permissions. This is not a problem since the permissions of symbolic links are never used.